lib/GTK/Simple/Entry.pm6

```diff
@@ -19,7 +19,7 @@ method text()
     is gtk-property(&gtk_entry_get_text, &gtk_entry_set_text)
     { * }
 
-method width($num-chars)
+method width-chars($num-chars)
```

Comment:
Does this break API? What is the motivation behind it?

Reply:
I made this change so that it a) reflected the underlying GtkProperty and b) hopefully to make it more clear that about the unit of width, which in a generic context might be confused for pixels.

I don't mind reverting it though :)

Reply:
I think your points are nice, but I'd preserve the old version, calling the new version, with `is deprecated` trait enabled so that existing code had at least some time to migrate.

Reply:
Ah, this whole property was only added in this PR :)

Reply:
That is to say, it's not a breaking change because this method hasn't been in the API before.

Reply:
Sorry for not being clearer from the beginning.. I misread your original question.

Reply:
I'm going to close this PR and re-open with a squashed commit.

Thanks for taking the time to check it out!

Reply:
> I'm going to close this PR and re-open with a squashed commit.
> Thanks for taking the time to check it out!
> EDIT: Found the squash merge option in the Github interface, so no need to re-submit

or you could push force a squash to your fork `git reset --soft HEAD~number_of_commits_to_squash; git push fork_origin branch --force`. Github will pick it up automatically

Reply:
but a big squashed commits sounds like a bad idea to me anyway